webview/src/plots/components/ribbon/RibbonBlock.tsx

```diff
@@ -47,6 +47,7 @@ export const RibbonBlock: React.FC<RibbonBlockProps> = ({
       style={{ borderColor: displayColor }}
       data-testid={`ribbon-${id}`}
     >
+      <Info className={styles.infoIcon} />
       <div className={styles.label}>
         {description ? (
           <>
```

Comment:
Noticed while testing things that copy buttons are broken in the ribbon tooltips. Will fix this!

Reply:
#4147